site/content/docs/manifest/lb-web-service.en.md

```diff
@@ -73,7 +73,63 @@ The name of your service.
 The architecture type for your service. A [Load Balanced Web Service](../concepts/services.en.md#load-balanced-web-service) is an internet-facing service that's behind a load balancer, orchestrated by Amazon ECS on AWS Fargate.
 
-{% include 'http-config.en.md' %}
+<div class="separator"></div>
+
+<a id="http" href="#http" class="field">`http`</a> <span class="type">Boolean or Map</span>  
+The http section contains parameters related to integrating your service with an Application Load Balancer.
+
+To disable the Application Load Balancer, specify `http: false`. Note that for a Load-Balanced Web Service,
+at least one of Application Load Balancer or Network Load Balancer must be enabled.
+
+<span class="parent-field">http.</span><a id="http-path" href="#http-path" class="field">`path`</a> <span class="type">String</span>  
+Requests to this path will be forwarded to your service. Each Load Balanced Web Service should listen on a unique path.
+
+{% include 'http-healthcheck.en.md' %}
+
+<span class="parent-field">http.</span><a id="http-deregistration-delay" href="#http-deregistration-delay" class="field">`deregistration_delay`</a> <span class="type">Duration</span>  
+The amount of time to wait for targets to drain connections during deregistration. The default is 60s. Setting this to a larger value gives targets more time to gracefully drain connections, but increases the time required for new deployments. Range 0s-3600s.
+
+<span class="parent-field">http.</span><a id="http-target-container" href="#http-target-container" class="field">`target_container`</a> <span class="type">String</span>  
+A sidecar container that takes the place of a service container.
+
+<span class="parent-field">http.</span><a id="http-stickiness" href="#http-stickiness" class="field">`stickiness`</a> <span class="type">Boolean</span>  
+Indicates whether sticky sessions are enabled.
+
+<span class="parent-field">http.</span><a id="http-allowed-source-ips" href="#http-allowed-source-ips" class="field">`allowed_source_ips`</a> <span class="type">Array of Strings</span>  
+CIDR IP addresses permitted to access your service.
+```yaml
+http:
+allowed_source_ips: ["192.0.2.0/24", "198.51.100.10/32"]
+```
+
+<span class="parent-field">http.</span><a id="http-alias" href="#http-alias" class="field">`alias`</a> <span class="type">String or Array of Strings or Array of Maps</span>  
+HTTPS domain alias of your service.
+```yaml
+# String version.
+http:
+  alias: example.com
+# Alternatively, as an array of strings.
+http:
+  alias: ["example.com", "v1.example.com"]
+# Alternatively, as an array of maps.
+http:
+  alias:
+    - name: example.com
+      hosted_zone: Z0873220N255IR3MTNR4
+    - name: v1.example.com
+      hosted_zone: AN0THE9H05TED20NEID
+```
+<span class="parent-field">http.</span><a id="http-hosted-zone" href="#http-hosted-zone" class="field">`hosted_zone`</a> <span class="type">String</span>  
```

Comment:
We might need to remind them they can only use hosted zone when using alias? Also should we highlight this new feature [here](https://github.com/aws/copilot-cli/blob/a2cb81b05ba48b53a360fdbd4832cc50e1ac0a45/site/content/docs/developing/domain.en.md?plain=1#L77) as well (might be helpful for those users)?